drivers/scheduler/k8s/specs/elasticsearch-crd-webhook/pxd/px-storage-class.yaml

```diff
@@ -0,0 +1,12 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+    name: elasticsearch-sc
+provisioner: kubernetes.io/portworx-volume
```

Comment:
Can we use the CSI provisioner instead `pxd.portworx.com`?

Reply:
I don't know what that means, could you clarify? 
I just copied these files from another spec and modified them with the "right" details.

Reply:
✅